src/dashboard/src/components/archival_storage/views.py

```diff
@@ -186,32 +195,117 @@ def get_es_property_from_column_index(index, file_mode):
     return table_columns[file_mode][index]
 
 
+def _normalize_and_capitalize_csv_header():
+    """Capitalize the column headers of a Pandas DataFrame
+
+    Because the Archivematica code uses a mix of snake case, camel case
+    and not well formed variants of either plus all-caps we a dict
+    provides the lo-fi way to do this without changing significant parts
+    of indexing.
+
+    Columns are ordered so that the data structure can be reused to
+    guarantee an ordering during output.
+    """
+    column_lookup = OrderedDict()
+    column_lookup[es.ES_FIELD_NAME] = _("Name")
+    column_lookup[es.ES_FIELD_UUID] = _("UUID")
+    column_lookup[es.ES_FIELD_AICID] = _("AICID")
+    column_lookup[es.ES_FIELD_AICCOUNT] = _("Count AIPs in AIC")
+    column_lookup[es.ES_FIELD_SIZE] = _("Size")
+    column_lookup[es.ES_FIELD_FILECOUNT] = _("File count")
+    column_lookup[es.ES_FIELD_ACCID] = _("Accession IDs")
+    column_lookup[es.ES_FIELD_CREATED] = _("Created date (UTC)")
+    column_lookup[es.ES_FIELD_STATUS] = _("Status")
+    column_lookup["type"] = _("Type")
+    column_lookup[es.ES_FIELD_ENCRYPTED] = _("Encrypted")
+    column_lookup[es.ES_FIELD_LOCATION] = _("Location")
+    return column_lookup
+
+
+def search_as_csv(data_frame, file_name=CSV_FILE_NAME, mime_type=CSV_MIMETYPE):
+    if mime_type != CSV_MIMETYPE:
+        mime_type = CSV_MIMETYPE
+    ACCESSION_IDS_FIELD = es.ES_FIELD_ACCID
+    FIELD_SEPARATOR = "; "
+    ENCODING = "utf-8"
+    CONTENT_DISPOSITION_HDR = "Content-Disposition"
+    MIME_HDR = "mimetype"
+    CONTENT_DISPOSITION = 'attachment; filename="{}"'.format(file_name)
+    for i, cell in enumerate(data_frame[ACCESSION_IDS_FIELD]):
+        normalized_value = FIELD_SEPARATOR.join(cell)
+        data_frame.at[i, ACCESSION_IDS_FIELD] = normalized_value
+    CREATED_FIELD = es.ES_FIELD_CREATED
+    data_frame[CREATED_FIELD] = data_frame[CREATED_FIELD].apply(str)
+    for i, cell in enumerate(data_frame[CREATED_FIELD]):
+        normalized_value = make_aware(
+            datetime.fromtimestamp(float(cell)), timezone=get_current_timezone()
+        )
+        data_frame.at[i, CREATED_FIELD] = normalized_value
```

Comment:
The second option is quite attractive and complements the first too so I've combined both for the version I'll submit and I think it may help the code be more descriptive too. Nice work.